ui/src/app/shared/utils/url-search-params.utils.ts

```diff
@@ -154,7 +154,14 @@ import {TimeFrame} from "../model/TimeFrame";
     let queryFields = new Map<string, FieldDataType>(queryDataTypes);
 
     if (capabilities.commonLabels) {
-      capabilities.commonLabels.map(label => queryFields.set(label, FieldDataType.Text));
+      capabilities.commonLabels.map(label => {
```

Comment:
existing issue, but would use `forEach` if you're not consuming the return value of the `map()` call

Reply:
Ok, done